<ui version="4.0" >
 <class>ModifyPasswordClass</class>
 <widget class="QWidget" name="ModifyPasswordClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ModifyPassword</string>
  </property>
  <widget class="QLabel" name="currentUser" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>70</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>180</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>原 密 码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPasswordLineEdit" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>180</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>250</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>新 密 码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>320</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>250</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmLineEdit" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>320</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>确  定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
